modules/efk-logging-fluentd-external.adoc

```diff
@@ -51,10 +71,46 @@ $ oc edit configmap/fluentd -n openshift-logging
 ----
 <1> Enter the name, host, and port for your external Fluentd server.
 
+. Add the path to your certificate and private key to the `secure-forward.conf` section:
++
+[source,yaml]
+----
+  secure-forward.conf: |
+    # <store>
+    # @type secure_forward
```

Comment:
This config is for 3.11.  Use https://github.com/openshift/origin-aggregated-logging/blob/master/fluentd/configs.d/user/secure-forward.conf as the sample config for 4.x